dist/README.md

```diff
@@ -1,79 +1,79 @@
 # Deplyment of ovsdb-etcd on OVN kubernetes
 
-In order to deploy [OVN kubernetes](https://github.com/ovn-org/ovn-kubernetes) with ovsdb-etcd follow the following steps:
-
-## Build ovn
-
-```bash
-git clone https://github.com/hunchback/ovn
-cd ovn
-git checkout ovndb-etcd
-./boot.sh
-./configure
-make
-export OVN_ROOT=$PWD
-```
 
 ## Prepare ovs-kubernetes
 
 Clone OVN kubernetes from the main repository:
 
 ```bash
-# Clone ovn-kubernetes repo
 git clone https://github.com/hunchback/ovn-kubernetes
 cd ovn-kubernetes
-git checkout ovndb-etcd
+git checkout ovsdb-etcd2
 export OVN_KUBERNETES_ROOT=$PWD
 		```
-Build the OVN kubernetes container image:
-
-```bash
-make -C go-controller
-```
-
-```bash
-cp $OVN_ROOT/utilities/ovn-nbctl dist/images/
-make -C dist/images fedora
-```
-
-## Build ovn-kubernetes
+## Build docker images
 
-clone and build and push (aboute 5 minutes):
+clone and build and push:
```

Comment:
should we mention cd .. (or something similar) ? I mean not to clone ovsdb-etcd into ovn-kubernetes